### PR TITLE
V9 to v10 upgrade patch

### PR DESCRIPTION
**Pull request by jonat123**
I have updated the docs for upgrading from 9 to 10 and made the different steps clearer.

### CODE REVIEW

File: Fundamentals/Setup/Upgrading/version-specific.md [hunk 1]
```diff
@@ -9,15 +9,27 @@ versionFrom: 7.0.0
...
+- Go to "__tools > NuGet Package Manager > Manage NuGet Packages for Solutions...__"
+
+- In the NuGet Package manager go to "Installed" and choose __Umbraco.Cms__
+
+- Choose Umbraco 10 and install the version in your project.
+
+- Build and run your project to finish the installation of Umbraco 10.
+
+After updating the project through NuGet, you will need update your project from `net5.0` to `net6.0`
+Additionally, you will need to update the `Program.cs`to the following:
```
Comment (jonat123) [inline Fundamentals/Setup/Upgrading/version-specific.md:28]:
```suggestion
- Go to "__tools > NuGet Package Manager > Manage NuGet Packages for Solutions...__"

- In the NuGet Package manager go to "Installed" and choose __Umbraco.Cms__

- Choose Umbraco 10 and install the version in your project.

- Build and run your project to finish the installation of Umbraco 10.

After updating the project through NuGet, you will need to update your project from `net5.0` to `net6.0`
Additionally, you will need to update the `Program.cs` to the following:
```

File: Fundamentals/Setup/Upgrading/version-specific.md [hunk 2]
```diff
@@ -30,14 +42,14 @@ We updated the `Program` class in the `Program.cs` file to the following out of
...
-                    webBuilder.UseStartup&lt;Startup&gt;();
+                    webBuilder.UseStartup<Startup>();
                 });
     }
 ```
 
 The calls to `ConfigureUmbracoDefaults` and `webBuilder.UseStaticWebAssets()` are new.
 
-Furthermore, it is recommended to remove the following files and folders:
+Finally remove the following files and folders:
```
Comment (jonat123) [inline Fundamentals/Setup/Upgrading/version-specific.md:52]:
```suggestion
Finally, remove the following files and folders:
```

File: Fundamentals/Setup/Upgrading/version-specific.md [hunk 3]
```diff
@@ -9,15 +9,27 @@ versionFrom: 7.0.0
...
-After updating the nuget dependency, you will need update your project from `net5.0` to `net6.0`
-Additionally, you will need to update `Program.cs` and remove some folders from your solution.
+The upgrade path between Umbraco 9 and Umbraco 10 can be done directly by updating your project using NuGet.
 
-We updated the `Program` class in the `Program.cs` file to the following out of the box:
+### Steps on how to upgrade using Visual Studio
+
+- Open your Umbraco 9 project in Visual Studio
+
+- Go to "__tools > NuGet Package Manager > Manage NuGet Packages for Solutions...__"
```
Comment (eshanrnh) [inline Fundamentals/Setup/Upgrading/version-specific.md:19]:
```suggestion
- Go to "__Tools > NuGet Package Manager > Manage NuGet Packages for Solution...__"
```

File: Fundamentals/Setup/Upgrading/version-specific.md [hunk 4]
```diff
@@ -9,15 +9,27 @@ versionFrom: 7.0.0
...
-The upgrade path between Umbraco 9 and Umbraco 10 can be done directly by updating your project NuGet references.
 
-After updating the nuget dependency, you will need update your project from `net5.0` to `net6.0`
-Additionally, you will need to update `Program.cs` and remove some folders from your solution.
+The upgrade path between Umbraco 9 and Umbraco 10 can be done directly by updating your project using NuGet.
 
-We updated the `Program` class in the `Program.cs` file to the following out of the box:
+### Steps on how to upgrade using Visual Studio
+
+- Open your Umbraco 9 project in Visual Studio
```
Comment (eshanrnh) [inline Fundamentals/Setup/Upgrading/version-specific.md:17]:
```suggestion
- Open your Umbraco 9 project in Visual Studio.
```

File: Fundamentals/Setup/Upgrading/version-specific.md [hunk 5]
```diff
@@ -9,15 +9,27 @@ versionFrom: 7.0.0
...
+
+- Go to "__tools > NuGet Package Manager > Manage NuGet Packages for Solutions...__"
+
+- In the NuGet Package manager go to "Installed" and choose __Umbraco.Cms__
```
Comment (eshanrnh) [inline Fundamentals/Setup/Upgrading/version-specific.md:21]:
```suggestion
- In the NuGet Package manager go to **Installed** and choose __Umbraco.Cms__
```

File: Fundamentals/Setup/Upgrading/version-specific.md [hunk 6]
```diff
@@ -9,15 +9,27 @@ versionFrom: 7.0.0
...
+
+- Choose Umbraco 10 and install the version in your project.
```
Comment (eshanrnh) [inline Fundamentals/Setup/Upgrading/version-specific.md:23]:
```suggestion
- Choose **10.0.0** from the **Version** drop-down and click **Install** to upgrade your project to version 10.
```

File: Fundamentals/Setup/Upgrading/version-specific.md [hunk 7]
```diff
@@ -30,14 +42,14 @@ We updated the `Program` class in the `Program.cs` file to the following out of
                 .ConfigureWebHostDefaults(webBuilder =>
                 {
                     webBuilder.UseStaticWebAssets();
-                    webBuilder.UseStartup&lt;Startup&gt;();
+                    webBuilder.UseStartup<Startup>();
```
Comment (eshanrnh) [inline Fundamentals/Setup/Upgrading/version-specific.md:45]:
```suggestion
                    webBuilder.UseStartup&lt;Startup&gt;();
```
Our doesn't display the <Startup> as it considers it a code and skips it. To display it in the document, it needs to be written as &lt;Startup&gt; 

See issue https://github.com/umbraco/UmbracoDocs/issues/3956 where I had resolved this as a solution 🙂

File: Fundamentals/Setup/Upgrading/version-specific.md [hunk 8]
```diff
@@ -46,14 +58,12 @@ Furthermore, it is recommended to remove the following files and folders:
 - `/umbraco/UmbracoWebsite`
 - `/umbraco/config/lang`
 
-All these files are now served directly from a Razor Class Library (Umbraco.Cms.StaticAssets) or as embedded resources.
-
+All these files are now served directly from a Razor Class Library (__Umbraco.Cms.StaticAssets__) or as embedded resources.
 
 :::note
-Please note to upgrade to Umbraco 10, you need your database to be at least on Umbraco 8.18.
+Please note to upgrade to Umbraco 10, your database needs to be at least on Umbraco 8.18.
```
Comment (eshanrnh) [inline Fundamentals/Setup/Upgrading/version-specific.md:64]:
```suggestion
To upgrade to Umbraco 10, your database needs to be at least on Umbraco 8.18.
```

**Review by eshanrnh — COMMENTED** [review]
Looks good 💪 Just a couple of suggestions.

File: Fundamentals/Setup/Upgrading/version-specific.md (hunk 7 above)
Comment (jonat123) [reply Fundamentals/Setup/Upgrading/version-specific.md:45]: Ahhh okay, thanks for letting me know 🙂

**Review by eshanrnh — APPROVED** [review]
Merging it in 🙂

**Pull request merged**